tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "compilerOptions": {
     "outDir": "./dist/",
     "noImplicitAny": true,
-    "module": "es6",
+    "module": "es2020",
```

Comment:
I'd like to do just a bit more research into this, but my initial understanding is es2020 should be very close to es6 so this is probably fine. Let's add a comment here as to why (`es6 does not allow dynamic imports, which we use for React lazy() loading/code splitting`? These config files tend to become very opinionated in mysterious ways as time passes.